mumuki-laboratory.gemspec

```diff
@@ -16,9 +16,9 @@ Gem::Specification.new do |s|
 
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 5.1.6"
+  s.add_dependency "rails", "~> 5.2"
 
-  s.add_dependency 'mumuki-domain', '~> 6.6.1'
+  s.add_dependency 'mumuki-domain', '~> 99.99.99' # TODO: Update gem
```

Comment:
Instead of doing this, you can place a `gem github: ...., ref: ...` directive in the Gemfile so that you can work even against of an unreleased version. Then we remove those directives when releasing new versions. 

That way travis can pass and we can merge the PR even before we have a release candidate for the dependency. 

This is specially true when dealing with changes in domain, since we usually release domain, bibliotheca and laboratory altogether, using the same major and minor version.

Reply:
That's great to know! I'll use it